tools/compare.cpp

```diff
@@ -146,6 +150,7 @@ void printDifference(const Serializer& serializer1, const Serializer& serializer
 
 	std::cout << " | Number of values: " << std::setw(6) << nValues << "\n";
 	std::cout << " | Number of errors: " << std::setw(6) << nErrors << "\n";
+	if (nNan > 0) std::cout << " | Number of Nan errors: " << std::setw(6) << nNan << "\n";
```

Comment:
OK I'm going to correct it. What do I do? Close this PR and make a new one with NaN?